src/Controls/src/Core/TitleBar/TitleBar.iOS.cs

```diff
@@ -0,0 +1,27 @@
+using UIKit;
+using Microsoft.Maui.Platform;
+using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class TitleBar
+	{
+		internal override void OnIsVisibleChanged(bool oldValue, bool newValue)
+		{
+			base.OnIsVisibleChanged(oldValue, newValue);
+
+#if MACCATALYST
+            var mauiContext = Handler?.MauiContext;
+
+            var platformWindow = Window.Handler.PlatformView;
```

Comment:
Can use directly the MauiContext:
`var platformWindow = mauiContext?.GetPlatformWindow();`